card.css

```diff
@@ -0,0 +1,130 @@
+.card-container {
```

Comment:
content is overlapping in preview which you have provided.

Reply:
Preview is not right, my mistake should have added images of card.